src/components/Registration/HelpContactUs.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div id="help-contact-us">
+    <div class="help-btn" @click="helpToggle = !helpToggle">
+      <v-icon color="primary" class="mt-n1">mdi-help-circle-outline</v-icon>
+      <span v-if="!helpToggle" class="pl-2">Need Help? Contact Us</span>
+      <span v-else class="pl-2">Hide Help</span>
+    </div>
+
+    <v-expand-transition>
+      <section v-if="helpToggle" class="help-section">
+        <header class="help-header">
+          <h2 class="py-4 px-0">Contact BC Registries</h2>
+        </header>
+
+        <p class="my-4">
+          If you require further assistance adding a business or corporation, please contact us.
+        </p>
+
+        <RegistriesContactInfo />
+
+        <div class="help-btn bottom" @click="helpToggle = !helpToggle">Hide Help</div>
+      </section>
+    </v-expand-transition>
+  </div>
+</template>
```

Comment:
Just had to create something VERY similar although different 😆

Reply:
Yeah, this feature is all about "very similar although different". We do have a generic help panel component but this is just too different. Sigh.